README.md

```diff
@@ -14,7 +14,7 @@ This project is a replicate of figma mobile template one
 1. Setup
 2. Install
 3. Usage
-4. Tun tests
+4. Run tests
 5. Deployment
```

Comment:
[Optional] All of these are just headings which you're supposed to give descriptions for those that apply to the project you're working. Consider taking off those that don't apply to this project like `Run tests`